### PR TITLE
file selector package for desktop

### PR DESCRIPTION
**Pull request by josuecapgemini**

### CODE REVIEW

File: wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart [hunk 1]
```diff
@@ -234,4 +241,33 @@ class _EditProfilePageState extends State<EditProfilePage> {
           ])),
     );
   }
+
+  Widget _getImageFromCameraOrGalleryDesktop() {
```
Comment (capf4bbo) [inline wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart:244]: Testing for widget missing

File: wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart [hunk 2]
```diff
@@ -146,7 +147,13 @@ class _EditProfilePageState extends State<EditProfilePage> {
           } else {
             showModalBottomSheet(
                 context: context,
-                builder: (builder) => _getImageFromCameraOrGallary());
+                builder: (builder) {
+                  if (widget.platformService.isDesktop) {
```
Comment (capf4bbo) [inline wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart:150]: Maybe you can make a test to check which of these functions are called

File: wayat/frontend/lib/features/groups/pages/manage_group_page.dart [hunk 3]
```diff
@@ -408,4 +414,31 @@ class ManageGroupPage extends StatelessWidget {
           ])
         ]));
   }
+
+  Widget openSelectImageSheetDesktop(BuildContext context) {
```
Comment (capf4bbo) [inline wayat/frontend/lib/features/groups/pages/manage_group_page.dart:418]: Testing for the widget

**Review by capf4bbo — CHANGES_REQUESTED** [review]

File: wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart (hunk 1 above)
Comment (josuecapgemini) [reply wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart:244]: Test done.

File: wayat/frontend/lib/features/groups/pages/manage_group_page.dart (hunk 3 above)
Comment (josuecapgemini) [reply wayat/frontend/lib/features/groups/pages/manage_group_page.dart:418]: Test done.

File: wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart (hunk 2 above)
Comment (josuecapgemini) [reply wayat/frontend/lib/features/profile/pages/edit_profile_page/edit_profile_page.dart:150]: I tried it but it gives me an error, so I decided to delete the test.

**Review by capf4bbo — APPROVED** [review]

**Pull request merged**